GrayBMP.cs

```diff
@@ -110,6 +110,35 @@ public void Dirty ()
       End ();
    }
 
+   public void DrawThickLine (int x0, int y0, int x1, int y1, int width, int color, PolyFillFast mPF) {
+      var leftVertices = new Point2[4];
+      var rightVertices = new Point2[4];
+      var factor = PI / 180;
+      int index = 0;
+      width /= 2;
+      var angleLine = Math.Atan2 ((y1 - y0), (x1 - x0));
+      for (int angle = 90; angle <= 270; angle += 60) {
+         var tmpAngle = angleLine + (angle * factor);
+         var x01 = (int)(x0 + (width * Cos (tmpAngle)));
+         var y01 = (int)(y0 + (width * Sin (tmpAngle)));
+         leftVertices[index] = (new (x01, y01));
+         var x11 = (int)(x1 + (-width * Cos (tmpAngle)));
+         var y11 = (int)(y1 + (-width * Sin (tmpAngle)));
+         rightVertices[index] = (new (x11, y11));
+         index++;
+      }
+      mPF.Reset ();
+      for (int i = 0; i < 3; i++) {
+         if (i == 0)
+            mPF.AddLine ((int)leftVertices[i].X, (int)leftVertices[i].Y, (int)rightVertices[3].X, (int)rightVertices[3].Y);

```

Comment:
Use smaller names like **left** and **right**, especially with often-repeated local variables, to avoid such cluttered code.